highfidelity/hifi-content
# Update to plates

**Test Plan** 
1. Pull a plate off the middle shelf in the hub
2. Confirm that the test plan from #117 remains in tact. A new plate should spawn within 5 seconds of removing the one on the shelf.
3. Test that the audio is now localized and the smaller pieces play a slightly different sound
